app/assets/javascripts/classes/reports/EditReportViewModel.ts

```diff
@@ -197,6 +217,8 @@ class EditSourceViewModel extends BaseSourceViewModel {
               "<ul class='list-group'>"+
               "<li class='list-group-item'><b>Browser rendered</b> - A URL is loaded in the browser and the rendered " +
               "contents of the page are taken as the generated report.</li>" +
+              "<li class='list-group-item'><b>Grafana</b> - Like browser-rendered, but tweaked to pull data from a Grafana Report panel" +
+              " (which are loaded asynchronously after page-load, hence needing the specialization).</li>" +
```

Comment:
I'm starting to think a better distinction would be to have an empty event field for Browser Rendered, leaving "Grafana" exactly the same but with an event to wait for specified.

Reply:
(discussed in person; I believe we agree that these need to be separate, at least for now)